internal/pkg/skuba/deployments/ssh/kubelet.go

```diff
@@ -41,8 +41,13 @@ func kubeletConfigure(t *Target, data interface{}) error {
 		if err := t.UploadFileContents("/usr/lib/systemd/system/kubelet.service.d/10-kubeadm.conf", assets.KubeadmService); err != nil {
 			return err
 		}
-		if err := t.UploadFileContents("/etc/sysconfig/kubelet", assets.KubeletSysconfig); err != nil {
-			return err
+
+		const kubelet_sysconfig = "/etc/sysconfig/kubelet"
+		if contents, _ := t.DownloadFileContents(kubelet_sysconfig); contents == "" {
+			// Upload file only when it doesn't alreayd exist.
+			if err := t.UploadFileContents(kubelet_sysconfig, assets.KubeletSysconfig); err != nil {
+				return err
+			}
```

Comment:
This is changing `skuba` semantics and skuba should be more verbose of this change. Like adding an `else` case saying that `/etc/sysconfig/kubelet` already exists and will not be touched.